Lib/dbm/__init__.py

```diff
@@ -164,6 +164,10 @@ def whichdb(filename):
     if len(s) != 4:
         return ""
 
+    # Check for SQLite3 header string.
+    if s16 == b"SQLite format 3\0":
```

Comment:
I know the current comparison works, but can we use `3\000` to be consistent with [SQLite docs](https://www.sqlite.org/fileformat.html#the_database_header):
```suggestion
    if s16 == b"SQLite format 3\000":
```